releases/release-4.0.14.md

```diff
@@ -0,0 +1,135 @@
+---
+title: TiDB 4.0.14 Release Notes
+---
+
+# TiDB 4.0.14 Release Notes
+
+Release date: July 22, 2021
+
+TiDB version: 4.0.14
+
+## Compatibility Changes
+
++ TiDB
+
+    - Change the default value of `tidb_multi_statement_mode` from `WARN` to `OFF` in v4.0. It is recommended to use the multi-statement feature of your client library instead. See [the documentation on `tidb_multi_statement_mode`](/system-variables.md#tidb_multi_statement_mode-new-in-v4011) for details. [#25749](https://github.com/pingcap/tidb/pull/25749)
+    - Upgrade Grafana dashboard from v6.1.16 to v7.5.7 to solve two security vulnerabilities. See the [Grafana post](https://grafana.com/blog/2020/06/03/grafana-6.7.4-and-7.0.2-released-with-important-security-fix/) for details.
+
+## Improvements
+
```

Comment:
```suggestion

+ PD

    - Update TiDB Dashboard to v2021.07.17.1 [#3882](https://github.com/pingcap/pd/pull/3882)
    - Reduce the conflict due to multiple schedulers running at the same time [#3858](https://github.com/pingcap/pd/pull/3858) [#3854](https://github.com/tikv/pd/pull/3854)
    - Fix the issue that leader re-election be slow when there are many stores [#3718](https://github.com/pingcap/pd/pull/3718)

+ TiDB Dashboard

    - Support sharing session as read-only to avoid further modifications [#960](https://github.com/pingcap/tidb-dashboard/pull/960)

```